### PR TITLE
RomanBychkov homework24

### PR DESCRIPTION
**Pull request by RomanNoname**

### CODE REVIEW

File: Roman Bychkov/Lesson24/CalendarApp/CalindarTest/UnitTest1.cs [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+using CalendarApp.Contracts.Models;
+using CalendarApp.DAL.Repositories.Interfaces;
+using Moq;
+using System;
+using System.Collections.Generic;
+using Xunit;
+
+namespace CalindarTest
+{
+    public class UnitTest1
```
Comment (denys-kniaziev) [inline Roman Bychkov/Lesson24/CalendarApp/CalindarTest/UnitTest1.cs]: Name of test class should correspond to class that you test (ex. RoomServiceTests, ...)

File: Roman Bychkov/Lesson24/CalendarApp/StringUtils/UnitTest1.cs [hunk 2]
```diff
@@ -0,0 +1,47 @@
+using System.Linq;
+using Xunit;
+
+namespace StringUtils
+{
+    public class UnitTest1
```
Comment (denys-kniaziev) [inline Roman Bychkov/Lesson24/CalendarApp/StringUtils/UnitTest1.cs]: The same

**Review by denys-kniaziev — CHANGES_REQUESTED** [review]
Name of test project should be CalendarApp.BLL.Tests

**Comment by RomanNoname** [comment]
All done

**Review by denys-kniaziev — APPROVED** [review]

**Pull request merged**